### ISSUE TITLE
Question about normalization step in adversarial training example

### ISSUE THREAD

**Issue by linzzzzzz** [BorealisAI/advertorch#60]
Hello,

Thank you for the great work! I have a quick question.

I'm looking at the adversarial training example and there seems to be no normalization step for MNIST images. Is it on purpose?

Thanks

**Comment by gwding** [comment]
We found that for MNIST, whether having normalization does not seem to affect the performance of adversarially trained models on MNIST significantly, therefore we didn't add it.

If you would like normalization, then just add these lines after https://github.com/BorealisAI/advertorch/blob/master/advertorch_examples/tutorial_train_mnist.py#L53

```
from advertorch.utils import MNIST_MEAN
from advertorch.utils import MNIST_STD
from advertorch.utils import NormalizeByChannelMeanStd

model = nn.Sequential(NormalizeByChannelMeanStd(MNIST_MEAN, MNIST_STD), model)
```

**Comment by linzzzzzz** [comment]
Understood. Thank you for your quick reply!
